packages/now-client/src/utils/index.ts

```diff
@@ -81,17 +81,24 @@ export async function buildFileTree(
   path: string | string[],
   isDirectory: boolean,
   debug: Debug
-): Promise<string[]> {
+): Promise<{ fileList: string[]; ignoreList: string[] }> {
+  const ignoreList: string[] = [];
   let fileList: string[];
-  let { ig } = await getVercelIgnore(path);
+  let { ig, ignores } = await getVercelIgnore(path);
 
-  debug(`Found ${ig.ignores.length} rules in .vercelignore`);
+  debug(`Found ${ignores.length} rules in .vercelignore`);
   debug('Building file tree...');
 
   if (isDirectory && !Array.isArray(path)) {
     // Directory path
-    const cwd = process.cwd();
-    const ignores = (absPath: string) => ig.ignores(relative(cwd, absPath));
+    const ignores = (absPath: string) => {
+      const rel = relative(path, absPath);
+      const ignored = ig.ignores(rel);
+      if (ignored) {
+        ignoreList.push(rel);
+      }
+      return ignored;
+    };
     fileList = await readdir(path, [ignores]);
     debug(`Read ${fileList.length} files in the specified directory`);
```

Comment:
```suggestion
    debug(`Found ${fileList.length} files in the specified directory`);
```

Just a nit pick here 😄